### PR TITLE
Bug 1137904. Add headlines to MWC page.

### PR DESCRIPTION
**Pull request by jpetto**

### CODE REVIEW

**Comment by jpetto** [comment]
The "See all press" link is dependent upon #2783, but rest of code good for review.

@sgarrity r?

File: bedrock/firefox/templates/firefox/os/mwc-2015-preview.html [hunk 1]
```diff
@@ -65,6 +65,27 @@
     </div>
   </section>
 
+  <section id="news">
+    <div class="container">
+      <div class="left">
+        <h3>{{ _('In the news') }}</h3>
+        <a href="">{{ _('See all press') }}</a>
```
Comment (sgarrity) [inline bedrock/firefox/templates/firefox/os/mwc-2015-preview.html]: You mentioned this, but adding a note to ensure we remember to get the real link in here.

**Comment by sgarrity** [comment]
Markup and styles are r+

As soon as we have the "See all press" link, we're set.

**Comment by sgarrity** [comment]
Turns out the "See all press" link can be a simple link for now, since this page is en-US only. Replacing this PR with https://github.com/mozilla/bedrock/pull/2793

**Comment by sgarrity** [comment]
Thanks for the help with this @jpetto

**Pull request closed without merging**